Libraries/LibJS/Parser.cpp

```diff
@@ -697,12 +729,19 @@ NonnullRefPtr<BlockStatement> Parser::parse_block_statement()
     return block;
 }
 
-template<typename FunctionNodeType>
+template <typename FunctionNodeType>
```

Comment:
The project `.clang-format` file is not set up to make this change. Are you using a custom setting?

Reply:
Is there one available? I'm using my own settings, tweaked to look like Serenity style.

Reply:
Certainly, https://github.com/SerenityOS/serenity/blob/master/.clang-format :)

Reply:
it's...right there in the root of the repository...
I feel stupid now.